bin/desi_zcatalog

```diff
@@ -154,12 +172,15 @@ for ifile, rrfile in enumerate(redrockfiles):
             [Table(redshifts), Table(fibermap[fmcols]), Table(tsnr2[tsnr2cols])]
             ))
 
-    exp_fibermaps.append(expfibermap)
+    if "expfibermap" in locals():
```

Comment:
See comment above about explicitly setting `expfibermap=None` if that extension doesn't exist and then checking `if expfibermap is not None` here instead of leaving the variable undefined and checking in locals here.  Checking `if "expfibermap" in locals()` is similar, but slightly less robust in case someone ever added a `expfibermap` or `tsnr2` variable earlier in the code, not realizing that defining the variable (even as None or False) would suddenly break this.